usr/initiator.c

```diff
@@ -889,7 +883,7 @@ static void session_conn_error(void *data)
 	switch (error) {
 	case ISCSI_ERR_INVALID_HOST:
 		if (session_conn_shutdown(conn, NULL, ISCSI_SUCCESS))
-			log_error("BUG: Could not shutdown session.");
+			log_error("BUG: Could not shutdown session%d.", sid);
```

Comment:
"session%d" is inconsistent with use of "session:%d" previously

Reply:
done